app/scripts/services/mobileClients.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("MobileClientsService", function ($filter,
+    $q,
+    $uibModal,
+    $rootScope,
+    APIService,
+    DataService,
+    Logger,
+    NotificationsService) {
+
+    var mobileclientVersion = {
+      group: "mobile.k8s.io",
+      version: "v1alpha1",
+      resource: "mobileclients"
+    };
+    var remove = function (apiObject) {
+      if (!$rootScope.AEROGEAR_MOBILE_ENABLED) {
+        Logger("mobile is not enabled");
+        return;
+      }
+      var context = {
+        namespace: _.get(apiObject, 'metadata.namespace')
+      };
+      NotificationsService.hideNotification("remove-mobileclient-error");
+      DataService.delete(mobileclientVersion, apiObject.metadata.name, context, {
+        gracePeriodSeconds: 0
```

Comment:
Why is gracePeriodSeconds needed?

Reply:
happy to remove, was basing on an example I saw in the serviceinstance service

Reply:
Yeah, that's a special case. I wouldn't include that unless it's really needed.